scripts/solr_builder/Jenkinsfile

```diff
@@ -209,22 +208,33 @@ pipeline {
               when { expression { return params.INTO_SOLR_FROM_DUMP } }
               stages {
                 stage('Insert works & orphaned editions') {
+                  when { expression { return params.INDEX_WORKS_ORPHANS } }
                   steps {
-                    insert_into_solr('./index-type.sh work 5; ./index-orphans.sh', 6, '{works,orphans}_*', 'works-orphans-indexing.log.gz')
+                    insert_into_solr('./index-type.sh work 5 works-orphans; ./index-orphans.sh works-orphans', 6, 'works-orphans')
                   }
                 }
                 stage('Insert authors') {
-                  steps { insert_into_solr('./index-type.sh author 6', 6, 'authors_*', 'authors-indexing.log.gz') }
+                  when { expression { return params.INDEX_AUTHORS } }
+                  steps { insert_into_solr('./index-type.sh author 6 authors', 6, 'authors') }
                 }
               }
               post {
+                always {
+                  dir(env.HOST_SOLR_BUILDER_DIR) {
+                    // Save the docker logs as artifacts
+                    sh """for key in logs/work-orphans/*; do echo "\$key: "; docker logs \$key 2>&1; done | gzip > logs/works-orphans-indexing.log.gz"""
```

Comment:
@cdrini notes that saving logs in docker containers may be failing (doesn't make sense)

This may be a good candidate for a new Priority 3/4 issue